src/sydney-testnet/tokenomics.md

```diff
@@ -0,0 +1,132 @@
+# Tokenomics
+
+## Initial Total Supply
+
+- **100 Billion GGX**
+- **18 decimals**
+- **8886 Chain ID**
+
+All amounts in this document are in GGX.
+
+## APY
+
+Schedule of APY will be next:
+
+| Date | Yearly APY** |
+| -----|-----------------------|
+| 2024 |    16.00%             |
+| 2025 |    14.93%             |
+| 2026 |    13.93%             |
+| 2029 |    11.31%             |
+| 2044 |    4%                 |
+| 2054 |    2%                 |
+| 2055 |    2%                 |
+
+**APY decreases by 6.7% per year.**
+
+After 30 years, the APY stabilizes at 2%.
+
+### Current State
+
+- Initial total supply is distributed to multisig to distribute to selected network validators.
+- 18 decimals.
+- APY is configured to 16% and drops 6.7% every year until it reaches 2% (configured by `DefaultInflationPercent` and `DefaultInflationDecay`).
+- APY decrease happens every 365.25 days to address leap years.
+
+## Staking
+
+Each new validator enters `PoS` with `1100 GGX` staked for `1 year`.
+
+Staking rewards are distributed each session.
+
+- **Session period:** 4 hours
+- **Era period:** 4 hours
+
+### Slashing
+
+| Misbehavior | Amount | Parameters |
+| ------------| -------| ------------------ |
+| Consensus Offline | 0.01% to 44% * Stake | Starting from 10% validators offline, linear increase |  
+| Consensus Equivocation for Blocks (Double Sign) | 0.01% * Stake | Validator signs conflicting blocks |
+| MPC Signature (Wrong message) | 0.01% * Stake | Validator signs an incorrect MPC message |
+| MPC Key Generation (Failure to Participate) | 0.001% * Stake | Validator fails to participate in key generation |
+| Early Unstake | 10% * Stake | Unstaking before the bonding period ends |
+
+Slashed amounts are sent to the treasury.
+
+### Nominations
+
+| Account | Part | Type |
+| ------- | ---- | ---- |
+| Validator | Variable% | Configurable commission |
+| Nominator + Validator | Variable% | Shared between nominators and validator |
+
+- **Validators** receive a commission that is deducted from the nominators' rewards. The commission rate is configurable and determined as the median value of all validators' preferred rates. The remaining rewards, after the commission deduction, are shared between the nominators and the validator based on their respective stakes.
+- **10% of validators' rewards** go to the Treasury:
+
+| Account | Part |
+| --------- | ---- |
+| Validator | 90% |
+| Treasury | 10% |
+
+### Current State
+
+- 1-year stake withdrawal.
```

Comment:
@Liquid369 @xcthulhu Does it mean you can withdraw only in one year? Should we decrease it after one year?